composer.json

```
@@ -38,12 +40,12 @@
     },
     "autoload": {
         "psr-4": {
-            "Yiisoft\\_____\\": "src"
+            "Yiisoft\\DataDb\\": "src"
```

I'd adjust namespace:

```suggestion
            "Yiisoft\\Data\\Db\\": "src"
```